### ISSUE TITLE
Code Lab Feedback

### ISSUE THREAD

**Issue by zacharyweston** [zacharyweston/codelab-js-booleans-and-comparators#1]
Hey @chrisvfritz, can you take a look at this? It's [hosted here](http://equable-salt.surge.sh) and meets the following [project](http://lansingcodelab.com/lessons/js-booleans-and-comparators/1) criteria:
- [x] include at least 2 choices for the user to make
- [x] use `prompt` to collect choices from the user (the entire story can take place in dialog boxes, so your `body` element could be empty for this project)
- [x] use comparators with `if` and `else` to branch the story, depending on user choices
- [x] don't get _too_ carried away - it can be tempting to build a complex video game, but remember, you can always come back and build on this project once you learn more

**Comment by chrisvfritz** [comment]
Nice easter egg reference to Douglas Adams. :smiley: The JavaScript code looks great, but I do notice a couple HTML errors:
- [This first line](https://github.com/zacharyweston/codelab-js-booleans-and-comparators/blob/master/index.html#L1) declaring the doctype has a typo. The exclamation point comes _before_ `DOCTYPE` rather than after.
- Nothing should be _directly_ inside an `html` tag apart from the `head` and `body`. A `script` tag is a rare element that can actually go into either, though most typically at the very end of the `body`, so I would move your `script` tag there.

**Comment by zacharyweston** [comment]
Got it, so typically the script would just be in a separate file, right?  I updated the code and pushed it, and the new URL is http://uneven-level.surge.sh/

**Comment by chrisvfritz** [comment]
Yes, all the JavaScript in most websites will be in a separate file. For this lesson though - and especially considering that the JavaScript _is_ the app - inline is just fine.

And... everything looks perfect now! :shipit:
